exotic/api/ephemeris.py

```diff
@@ -415,8 +415,8 @@ def plot_periodogram(self, minper=0, maxper=0, minper2=0, maxper2=0):
         basis[1] = self.epochs
         basis[2] = np.sin(2 * np.pi * self.epochs / per)
         basis[3] = np.cos(2 * np.pi * self.epochs / per)
-        basis[4] = np.sin(4 * np.pi * self.epochs / per2)
-        basis[5] = np.cos(4 * np.pi * self.epochs / per2)
+        basis[4] = np.sin(2 * np.pi * self.epochs / per2)
```

Comment:
Consider forcing all the operands in this to float, e.g. 2 -> '2.' (Note the decimal after the numeral.) It may avoid any surprising rounding but moreover may also be more understandable to readers.

Reply:
Also, see others below. ...

Reply:
fixed